docs/resources/Channel.md

```diff
@@ -47,7 +47,7 @@ Represents a guild or DM channel within Discord.
 | ----------------------- | --- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
 | GUILD_TEXT              | 0   | a text channel within a server                                                                                                                       |
 | DM                      | 1   | a direct message between users                                                                                                                       |
-| GUILD_VOICE             | 2   | a voice channel within a server                                                                                                                      |
+| GUILD_VOICE             | 2   | a voice channel containing a chat within a server                                                                                                    |
```

Comment:
I'd be wary about the word "chat" here. Does "chat" have semantic meaning in the API / this documentation? If not, it might not even be worth mentioning that there's an associated text channel since all voice channels will have a text channel. The distinction will only mean anything to devs who used the API prior to the release, because after there will be no concept of a "voice channel without text".

Reply:
Well, I will remove that. Thank you!